Src/PerformanceCollector/Shared/Implementation/QuickPulse/Service contract/TelemetryDocument/EventTelemetryDocument.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace Microsoft.ManagementServices.RealTimeDataProcessing.QuickPulseService
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Runtime.Serialization;
+
+    [DataContract]
+    internal struct EventTelemetryDocument : ITelemetryDocument
+    {
+        [DataMember(EmitDefaultValue = false)]
+        public Guid Id { get; set; }
```

Comment:
This has been there from the beginning of full telemetry documents in QuckPulse - I planned to use it for internal QuickPulse-deduping originally. We ended up not using it, I'll drop it in the next stable version if it's still useless.